src/xml2js.coffee

```diff
@@ -169,6 +169,8 @@ class exports.Builder
             if typeof child is 'string' && @options.cdata && requiresCDATA child
               element = element.ele(key).raw(wrapCDATA child).up()
             else
+              if child === undefined || child === null
```

Comment:
You're right. I completely changed the line to `if not child?`, which works and catches both `null` and `undefined`.